test/info.py

```diff
@@ -62,6 +62,14 @@
     'lpc11u35_archlink_if': 0x9013,
 }
 
+# Maps a target application filename (sans extension) to the ID of the board 
+# it is built to run on. This mapping is also used to display a board name 
+# in the test output. Thus, name the target applications after the  board to
+# avoid confusing results.
+#
+# One-to-many is fine. Many-to-one is fine as long as only one (or none) target 
+# application is found on disk per connected board ID
+#
```

Comment:
OK. I pushed an updated comment. I took out the key/map relationship comment since it was written only with --targetdir in mind and I'm not sure what is allowed in the Compile API case. Not a big deal. I think 1:1 is what will mostly be used.